diff --git a/docroot/sites/all/modules/contrib/feeds/feeds_news/feeds_news.feeds_importer_default.inc b/docroot/sites/all/modules/contrib/feeds/feeds_news/feeds_news.feeds_importer_default.inc
index 3fa5ee7..555c817 100644
--- a/docroot/sites/all/modules/contrib/feeds/feeds_news/feeds_news.feeds_importer_default.inc
+++ b/docroot/sites/all/modules/contrib/feeds/feeds_news/feeds_news.feeds_importer_default.inc
@@ -40,30 +40,46 @@ function feeds_news_feeds_importer_default() {
             'source' => 'title',
             'target' => 'title',
             'unique' => FALSE,
+            'language' => 'und',
           ),
           1 => array(
             'source' => 'timestamp',
             'target' => 'created',
             'unique' => FALSE,
+            'language' => 'und',
           ),
           2 => array(
             'source' => 'url',
             'target' => 'url',
             'unique' => 1,
+            'language' => 'und',
           ),
           3 => array(
             'source' => 'guid',
             'target' => 'guid',
             'unique' => 1,
+            'language' => 'und',
           ),
           4 => array(
             'source' => 'description',
             'target' => 'field_feed_item_description',
+            'format' => 'full_html',
+            'field_language' => 'und',
+            'language' => 'und',
+          ),
+          5 => array(
+            'source' => 'url',
+            'target' => 'field_url:url',
             'unique' => FALSE,
           ),
         ),
         'input_format' => 'filtered_html',
         'author' => 0,
+        'authorize' => TRUE,
+        'insert_new' => 1,
+        'update_non_existent' => 'skip',
+        'skip_hash_check' => FALSE,
+        'language' => 'und',
       ),
     ),
     'content_type' => 'feed',
